src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/Snapshots/PackageRestoreConfiguredInput.cs

```diff
@@ -13,7 +13,7 @@ public PackageRestoreConfiguredInput(ProjectConfiguration projectConfiguration,
         {
             ProjectConfiguration = projectConfiguration;
             RestoreInfo = restoreInfo;
-            ConfiguredProjectVersion = configuredProjectVersion;
+            ConfiguredProjectVersion = (long)configuredProjectVersion;
```

Comment:
This cast could lead to problems in future. @lifengl can we assume the type of these versions?

Reply:
why just keep it IComparable, and never assumes it is long? we only compare it with another version number, which is IComparable anyway.